subworkflows/local/bam_variant_calling_haplotypecaller/main.nf

```diff
@@ -31,7 +31,13 @@ workflow BAM_VARIANT_CALLING_HAPLOTYPECALLER {
         // Add interval_name to allow correct merging with interval files
         .map{ meta, cram, crai, intervals, num_intervals -> [ meta + [ interval_name:intervals.baseName, num_intervals:num_intervals, variantcaller:'haplotypecaller' ], cram, crai, intervals, [] ] }
 
-    GATK4_HAPLOTYPECALLER(cram_intervals, fasta, fasta_fai, dict.map{ meta, dict -> [ dict ] }, dbsnp, dbsnp_tbi)
+    GATK4_HAPLOTYPECALLER(
+        cram_intervals,
+        fasta.map{ it -> [[:], it] },
+        fasta_fai.map{ it -> [[:], it] },
+        dict,
+        dbsnp.map{ it -> [[:], it] },
+        dbsnp_tbi.map{ it -> [[:], it] })
```

Comment:
can you modify the channels in the upstream subworkflow?

Reply:
Yeah, I was expecting some comment on that 😄 Let me see if I can fix it in the subworkflow.

Reply:
@maxulysse : You want it around [here](https://github.com/nf-core/sarek/blob/2b3517a2567fb9443682ffd498844c52ab66ed33/subworkflows/local/bam_variant_calling_germline_all/main.nf#L129-L135)?

Reply:
exactly, look at how we do it for CNVKIT for example